### PR TITLE
console: trim role names when submitting the form (close #5553)

### LINKED ISSUE #5553: Role names without white spaces
Hey folks!

Whenever I create a new Role under the Permissions tab, a new role. I cannot create a role name with white spaces. For example, `Sales Manager`
Interestingly, I made that type of role name before. To fix that I usually copy `Sales Manager` to an input field and then change words separately. For example, `Sales Manager` becomes `Sales Moderator`.

Thank you

### PR DESCRIPTION
**Pull request by abhi40308**
Earlier, when creating a role in permissions, we could not give whitespaces, as whitespaces were getting trimmed. This pull request fixes that issue.

### Description

fix #5553 

### Changelog (TODO)

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [x] Console

### Steps to test and verify

Go to data section, select a table and select permissions, create a role with trailing whitespaces. For example ` admin role  `.

### CODE REVIEW

File: console/src/components/Services/Data/TablePermissions/Actions.js [hunk 1]
```diff
@@ -723,7 +723,7 @@ const permChangePermissions = changeType => {
     const limitEnabled = permissionsState.limitEnabled;
 
     const table = permissionsState.table;
-    const role = permissionsState.role;
+    const role = permissionsState.role.replace(/ /g, '');
```
Comment (beerose) [inline console/src/components/Services/Data/TablePermissions/Actions.js:726]:
```suggestion
    const role = permissionsState.role.trim();
```

**Review by beerose — CHANGES_REQUESTED** [review]

File: console/src/components/Services/Data/TablePermissions/Actions.js (hunk 1 above)
Comment (beerose) [reply console/src/components/Services/Data/TablePermissions/Actions.js:726]: Doing replace whitespaces won't solve the original issue.

**Review by beerose — APPROVED** [review]
Console

**Review by tirumaraiselvan — APPROVED** [review]
A changelog might have been good since it's a user reported issue so it's definitely user-facing ;)

But, it's also very trivial so it's okay.

**Pull request merged**